File: examples/javascript/fibonacci.js
function fib(n) {
    let a = 0;
    let b = 1;
    let i = 0;
    while (i < n) {
        const c = a + b;
        a = b;
        b = c;
        i = i + 1;
    }
    return a;
}

let i = 0;
while (i < 10) {
    console.log(fib(i));
    i = i + 1;
}

File: examples/lua/fibonacci.lua
function fib(n)
  local a = 0
  local b = 1
  local i = 0
  while i < n do
    local c = a + b
    a = b
    b = c
    i = i + 1
  end
  return a
end

local i = 0
while i < 10 do
  print(fib(i))
  i = i + 1
end
